src/styles/blocks.scss/header.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
```

Comment:
You can use mixin or at least variable here 
https://sass-lang.com/documentation/at-rules/mixin